pkg/api/entries.go

```diff
@@ -438,3 +439,35 @@ func SearchLogQueryHandler(params entries.SearchLogQueryParams) middleware.Respo
 
 	return entries.NewSearchLogQueryOK().WithPayload(resultPayload)
 }
+
+// Attempt to retrieve a UUID from a backend tree
+// TODO: don't return a bool that specifies whether it is an error, but instead check
+// the middleware.Responder to see if it is an error type (how?)
+func AttemptRetrieveUUID(params entries.GetLogEntryByUUIDParams, uuid string, tid int64) (middleware.Responder, bool) {
```

Comment:
nit: I would just call this `RetrieveUUID`,  it's implied that we're attempting to do so

Reply:
That's fair :)